File: csharp/CAPIExamples/morphology_complete.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CBinding;
using System.Web.Script.Serialization;

namespace CAPIExamples
{
    class morphology_complete
    {
        static void Main()
        {
            //Example code to call Rosette API to get the complete set of morphological analysis results for a piece of text.
            CAPI MorphologyCAPI = new CAPI("your API key");
            try
            {
                Dictionary<string, Object> MorphologyResult = MorphologyCAPI.Morphology("The quick brown fox jumped over the lazy dog. Yes he did.");
                Console.WriteLine(new JavaScriptSerializer().Serialize(MorphologyResult));
            }
            catch (RosetteException e)
            {
                Console.WriteLine("Error Code " + e.Code.ToString() + ":" + e.Message);
            }
        }
    }
}

File: nodejs/examples/morphology_complete.js
/*
 * Example code to call Rosette API to get the complete set of morphological analysis
 * results for a piece of text.
 */

"use strict";

var Api = require("rosette-api").Api;
var ArgumentParser = require("argparse").ArgumentParser;
var DocumentParameters = require("rosette-api").DocumentParameters;

var parser = new ArgumentParser({
  addHelp: true,
  description: "Get the complete morphological analysis of a piece of text"
});
parser.addArgument(["--key"], {help: "Rosette API key", required: true});
var args = parser.parseArgs();

var docParams = new DocumentParameters();
var content = "${morphology_complete_data}";
docParams.setItem("content", content);

var api = new Api(args.key);
api.morphology(docParams, null, function(err, res) {
  if (err) {
    throw err;
  }
  else {
    console.log(res);
  }
});
